nfc/libs/NfcCoreLib/lib/Nci/phNciNfc_Init.c

```diff
@@ -42,6 +51,9 @@ phNciNfc_SequenceP_t gphNciNfc_NfccResetSequence[] = {
     {NULL, &phNciNfc_CompleteNfccResetSequence}
 };
 
+/** Core Reset notification timeout */
+#define PHNCINFC_CORE_RESET_NTF_TIMEOUT        (30)
+
```

Comment:
Changed into PHNCINFC_CORE_RESET_NTF_TIMEOUT_MS.
30 ms is a timeout based with our NFC Controller behaviour. After holding 10ms the chip in reset,
a Core Reset Notification will be generated about 5 ms seconds later.
As far as i know there is not timeout defined in the NCI specification (?).

Let me know if you see any further information on this item.